modules/farm_fd2/src/entrypoints/transplanting/transplanting.soilDisturbance.e2e.cy.js

```diff
@@ -0,0 +1,66 @@
+describe('Transplanting: Soil Disturbance Component', () => {
+  beforeEach(() => {
+    cy.restoreLocalStorage();
+    cy.restoreSessionStorage();
+
+    cy.login('admin', 'admin');
+    cy.visit('fd2/transplanting/');
+    cy.waitForPage();
+  });
+
+  afterEach(() => {
+    cy.saveLocalStorage();
+    cy.saveSessionStorage();
+  });
+
+  it('Soil disturbance accordion exists, is visible, is collapsed', () => {
+    cy.get('[data-cy="transplanting-soil-disturbance-accordion"]').should(
+      'be.visible'
+    );
+    cy.get('[data-cy="transplanting-soil-disturbance-accordion-title"]')
+      .should('be.visible')
+      .should('contain.text', 'Equipment & Soil Disturbance');
+    cy.get('[data-cy="transplanting-soil-disturbance-accordion-item"]').should(
+      'not.be.visible'
+    );
+  });
+
+  it('Soil disturbance accordion expands to show equipment selector', () => {
+    cy.get(
```

Comment:
Check if this needs to be clicked twice? If not remove one of the `click` calls.